docs/configuration/configuration.rst

```diff
@@ -4,10 +4,262 @@
+
+* driver_type; Which driver to configure the Scheduler with.
+
+  As we mention throughout this document, the Scheduler CSC can operate with different types of Scheduling Algorithms.
+  These algorithms are implemented as subclasses to the base :py:class:`Driver <lsst.ts.scheduler.driver.Driver>` class.
+  This parameter allows the user to select which driver to load.
+  By default the Scheduler CSC implements a set of supported drivers, but this parameter can also point to externally provided options.
+
+  The input is in the format of the module path, e.g.; ``lsst.ts.scheduler.driver.feature_scheduler`` will search and import a subclass of :py:class:`Driver <lsst.ts.scheduler.driver.Driver>` in this module.
+
+  For a list of supported drivers see :ref:`the module API <lsst.ts.scheduler.api>`.
+
+  For information on developing new drivers see the :ref:`Developer_Guide`.
+
+* startup_type; The method used to startup the scheduler.
+
+  The Scheduler has a couple different startup types that defines how the intenal state of the scheduling algorithm is constructed/re-constructed.
+
+  The available options are:
+
+  * HOT; hot start.
+
+    In this mode the scheduler loads a snapshot of a previously executed and stored scheduler state.
+    The input (see startup_database furthermore) will be similar to the files stored in the LFOA as mentioned above.
+
+  * WARM; worm start.
+
+    In this mode, the scheduler will start a driver with its initial empty setting, load a set of observations from a database and play those observations into the driver, like it would if the observations where being executed.
+    
+  * COLD; cold start (default).
+
+    In this mode the scheduler will only initiate the driver and will not playback any observation.
+ 
+* startup_database; Path to a file holding scheduler state or observation database to be used on HOT or WARM start.
+
+    The Scheduler CSC don't know how to load these databases.
+    The process is entirely handled by the selected driver and the Scheduler is only responsible for initializing the driver and call appropriate methods, passing in this information.
+
+* mode; The mode of operation of the scheduler.
+
+  The scheduler CSC has a couple different modes of operations that defines how it interacts with the ScriptQueue, the telemetry stream and the scheduling algorithm.
+
+  The available options are:
+
+  * SIMPLE;
+
+    The simple mode of operation causes the scheduler to deal with one target at a time.
+    When operating in this mode the Scheduler will;
+    1. gather telemetry,
+    2. update the driver with the most recent telemetry,
+    3. request one target,
+    4. send the target to the ScriptQueue for execution,
+    5. wait for the script execution to complete,
+    6. if the observation executes successfully, register the observation in the driver.
+
+  * ADVANCE;
+
+    The advance mode of operation is the one intended for real operation.
```

Comment:
Not sure if the mode `ADVANCE` can be renamed, but in this sentence `advance` -> `advanced`